### ISSUE TITLE
2025-12-19: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#526]
# 每日更新报告（2025-12-19）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-12-19 23:33:22 | Jinzhen Lin | [Quantization] fix marlin w8a8 check (#30961) |
| 2025-12-19 22:16:09 | Shanshan Shen | [CustomOp][Refactor] Extract common methods for ApplyRotaryEmb CustomOp (#31021) |
| 2025-12-19 21:39:54 | Thomas Parnell | [Bugfix] [Kernel] Triton attention kernels: mask out V blocks that fall outside sliding window (#30887) |
| 2025-12-19 20:26:35 | Nishidha Panpaliya | [CPU][Bugfix] Fix ppc64le CPU build (#30871) |
| 2025-12-19 20:16:18 | Li, Jiang | Enable aarch64 CPU performance benchmarks (#26494) |
| 2025-12-19 18:47:44 | Marko Rosenmueller | [Frontend][Bug] allow tool calls in analysis channel (#28139) |
| 2025-12-19 18:23:28 | lif | [Bugfix] Add validation for tool requests when tool_parser is unavailable (#30613) |
| 2025-12-19 16:56:35 | Jinzhen Lin | [Quantization] enable compressed-tensors marlin support for turing (2) (#31008) |
| 2025-12-19 16:00:33 | Isotr0py | [Bugfix] Fix incorrect tiles creation for mm prefix triton attention (#30974) |
| 2025-12-19 15:59:36 | Wenqi Glantz | Add hidden dimension validation for multimodal embedding inputs (#30968) |
| 2025-12-19 15:29:52 | Li, Jiang | [Doc][CPU] Fix index link for CPU regular release wheels (#31015) |
| 2025-12-19 12:29:48 | Jinzhen Lin | [Quantization] enable compressed-tensors marlin support for turing (#31000) |
| 2025-12-19 11:58:28 | Nick Hill | [BugFix] Fix logprobs with spec decode and modified logits (#30846) |
| 2025-12-19 10:17:27 | Andreas Karatzas | [ROCm][CI][Bugfix] Multi-Modal Model Support Fixes and Attention Backend Improvements (#30270) |
| 2025-12-19 09:34:27 | PlatinumGod | [Bugfix] Fix tool_choice="none" being ignored by GPT-OSS/harmony models (#30867) |
| 2025-12-19 09:29:11 | Nick Hill | [BugFix] Handle errors when preprocessing added requests (#30895) |
| 2025-12-19 06:45:30 | Benjamin Chislett | [Cleanup] Refactor FlashInferMetadataBuilder (#29128) |
| 2025-12-19 06:10:02 | Chendi.Xue | [NIXL][BUG FIX] Fix both failing issue and accuracy issue with nixl + host_buffer on CUDA (#30419) |
| 2025-12-19 05:59:10 | Harry Mellor | Check for truthy `rope_parameters` not the existence of it (#30983) |
| 2025-12-19 04:59:55 | Nick Hill | [BugFix] Fix spec decode + structured outputs + preemption edge case (#30916) |
| 2025-12-19 03:51:58 | Kayvan Mivehnejad | Strengthen input validation and tests for 'parse_raw_prompts’. (#30652) |
| 2025-12-19 03:47:56 | Chen Zhang | [BugFix] Spec decode with VLLM_ENABLE_V1_MULTIPROCESSING=0 (#30319) |
| 2025-12-19 03:46:28 | Elizabeth Thomas | Remove all2all backend envvar (#30363) |
| 2025-12-19 03:45:55 | Wentao Ye | [Bug] Fix compressed tensor not using deepgemm (#30820) |
| 2025-12-19 03:42:32 | Isotr0py | [Bugfix] Remove `tile_size=64` for mm_prefix triton attention (#30973) |
| 2025-12-19 03:41:59 | navmarri14 | tuned fused configs for B300 (#30629) |
| 2025-12-19 03:31:18 | jiahanc | [Perf] enable flashinfer rotary_embedding custom ops in DeepSeek rotary (#30729) |
| 2025-12-19 03:09:51 | wz1qqx | [BugFix]Reclaim resources to prevent memory leaks when use LMCacheMPConnector (#30745) |
| 2025-12-19 02:16:36 | Fanli Lin | [XPU] allow custom workers (e.g. vllm-omni workers) to be used on XPU (#30935) |
| 2025-12-19 02:04:19 | Isotr0py | [MM Encoder]: Migrate legacy ViT `MultiHeadAttention` to new `MMEncoderAttention` interface (#30684) |
| 2025-12-19 01:52:55 | Alec | [BugFix] Add sleep to fix tight loop and release GIL (#29476) |
| 2025-12-19 01:50:42 | inkcherry | [Bugfix] fix DP-aware routing in OpenAI API requests (#29002) |
| 2025-12-19 01:17:00 | Nick Hill | [Cleanup] Remove unused ModelRunner V1 `InputBatch.num_tokens` field (#30218) |
| 2025-12-19 00:45:15 | Vasiliy Kuznetsov | fix fp8 online quantization streaming with tp > 1 (#30900) |
| 2025-12-19 00:42:22 | Xin Yang | [LoRA] Set default MXFP4 LoRA backend to Marlin (#30598) |
| 2025-12-19 00:29:33 | wzyrrr | [Doc] Add Sophgo TPU Support (#30949) |
| 2025-12-19 00:23:31 | Lucas Kabela | [Bugfix][torch2.10] Fix test_qwen2_5_vl_compilation with 2.10 RC (#30822) |
